dbt_style_guide.md

```diff
@@ -6,87 +6,148 @@ Our models (typically) fit into three main categories: staging, marts, base/inte
 ├── dbt_project.yml
 └── models
     ├── marts
-    |   └── core
-    |       ├── intermediate
-    |       |   ├── intermediate.yml
-    |       |   ├── customers__unioned.sql
-    |       |   ├── customers__grouped.sql
-    |       └── core.yml
-    |       └── core.docs
-    |       └── dim_customers.sql
-    |       └── fct_orders.sql
+    |   ├── core
+    |   |   ├── _docs.md

```

Comment:
do we want this to be more specific? `_core_docs.md`? do we have a convention for what to name a markdown file that has shared definitions used across many models?

Reply:
Yeah I originally was fine with _docs.md and then realized I had multiple files open with the same name. I am a fan of _<subdirectory>__docs.md

Reply:
I'm also a fan of context-specific naming, since the Cloud IDE is not always great at delineating which `_docs.yml` file have open. For consistency’s sake, I'd probably offer up `_docs_{subdirectory}.yaml` as an alternative. E.g. `_docs_core.yml`.

Reply:
👍 In the IDE when you have a specific file in focus in the editor, the tree highlights where the file is. However, I like this as it’s more apparent without having to know that!

Reply:
Do we want to do the same with `_models.yml` -> `_models_core.yml`? And `intermediate.yml` -> `_models_core_intermediate.yml`?